### DISCUSSION TITLE
Symfony get host cloudfront instead of xxx.execute-api.eu-west-3.amazonaws.com

### DISCUSSION THREAD

**Discussion by stloc** [brefphp/bref#1830] [Support]
$event->getRequest()->getSchemeAndHttpHost()
return http://xxx.execute-api.eu-west-3.amazonaws.com/ instead of lift cloudfront as https://dxdxdxxxx.cloudfront.net (no custom domain), how get cloudfront url ?

**Comment by mnapoli** [comment]
Can you check that you have this in place: https://bref.sh/docs/symfony/getting-started#trust-api-gateway ?

If that's the solution maybe we need to reword/improve the docs just a little bit?

**Reply by stloc** [reply]
Yes I implemented this configuration

**Reply by mnapoli** [reply]
Interesting, it might be worth dumping the headers to try and debug. Lift should configure CloudFront to forward the host: https://github.com/getlift/lift/blob/master/src/constructs/aws/ServerSideWebsite.ts#L383-L394 so something is not working correctly it seems.

**Reply by stloc** [reply]
OK thanks
